src/battle_command.c

```diff
@@ -1971,7 +1971,7 @@ BOOL BtlCmd_ChangeMonDataByVar(BattleSystem *bsys, BATTLECONTEXT *ctx) {
     }
 
     if (operator != 17) {
-        if (unkB == 26) {
+        if (unkB == BMON_DATA_ABILITY) {
```

Comment:
can unkB be also named here?

Reply:
yup, just did a quick double check and found 2 additional functions we can document too